ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressInput.component.ts

```diff
@@ -237,15 +243,23 @@ export class AddressInputComponent implements OnInit, OnDestroy {
     const valueChangedEmit$ = this.ais.addressOutputStream$.pipe(
       tap(val => this.valueChanged.emit(val)),
     );
+
+    const formValidStatusChangedEmit$ = merge(this.ais.inputAddress$, this.ais.addressForm.valueChanges).pipe(
+          tap(val => this.formValidStatusChanged.emit(this.ais.addressForm.valid))
+    );
+
```

Comment:
Not enough to emit on addressForm.valueChanges . Sometimes we set the form values from parent component with the emit event=false flag, so we need to merge that event here as well.